lib/layout/dummyLayoutV2.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * dummylayoutV2.js
+ *
+ * this creates a basic poorly laid out graph
+ *
+ * @author John O'Brien.
+ */
+
+var Node = require('../model/node.js');
+var Graph = require('../model/graph.js');
+var Port = require('../model/port.js');
+
+// Create a graph instance.
+
+function buildGraph(graphs) {
+  var nodes = graph.nodes;
+  //loop for building the nodes
+  for(var i = 0; i < nodes.length; i++){
+    nodes[i].x = i * 2;
+    nodes[i].y = i * 2;
+	
+    var ins = nodes[i].getIns();
+    var outs = nodes[i].getOuts();
+	
+    for(var numIns = 0; numIns < ins.Length; numIns++){
+      ins[ numIns ].setLocation("E", 0.5);
+    }
+	
+    for(var numOuts = 0; numOuts < outs.Length; numOuts++){
```

Comment:
Also, s/Length/length/. @johnwobrien, did you even run this? I don't see how it could have run successfully with a problem like this.